templates/shared/_thank_you_footer.html

```diff
@@ -1,26 +1,61 @@
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-10">
-      <h2>More about our enterprise services</h2>
-    </div>
+<section class="p-section--deep">
+  <div class="u-fixed-width p-section--shallow">
+    <hr />
+    <h2>More about our enterprise services</h2>
   </div>
-  <div class="row p-divider">
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Fully managed infrastructure</h3>
-      <p>Canonical offers fully managed infrastructure, including Kubernetes, OpenStack, Ceph and SWIFT storage and the recommended LMA stack.</p>
-      <p><a href="/pricing/infra#managed">Managed open infrastructure&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Consulting and deployment</h3>
-      <p>Open infrastructure consulting and training is also provided by Canonical, with optimal and custom architecture design and deployment.</p>
-      <p><a href="/pricing/consulting">Consulting, deployment from Canonical&nbsp;&rsaquo;</a></p>
-      <p><a href="/training">OpenStack, K8s, Kubeflow consulting &amp; training&nbsp;&rsaquo;</a></p>
-    </div>
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Infrastructure support &amp; security</h3>
-      <p>Access critical security fixes, legal assurance and phone and ticket support for OpenStack, K8s, Docker, Ceph and more through UA Infrastructure.</p>
-      <p><a href="/pricing/infra">Security maintenance and support&nbsp;&rsaquo;</a></p>
-      <p><a href="https://buy.ubuntu.com/">Purchase from our store</a></p>
+  <div class="row--25-75-on-large">
+    <div class="col">
+      <hr class="is-muted" />
```

Comment:
```suggestion
      <hr class="is-muted u-hide--small u-hide--medium" />
```
Fixes this double hr on smaller screens
![image](https://github.com/user-attachments/assets/355acd6c-6f96-4735-9d43-632ec20df31f)